src/components/data/UploadMenuBtn.js

```diff
@@ -15,16 +15,24 @@ import UploadDialog from "../uploads/dialog";
 
 import ids from "./ids";
 import styles from "./styles";
+import { useUploadTrackingState } from "../../contexts/uploadTracking";
 
 const useStyles = makeStyles(styles);
 
 function UploadMenuBtn(props) {
     const { baseId } = props;
     const classes = useStyles();
+    const tracker = useUploadTrackingState();
 
     const [uploadAnchor, setUploadAnchor] = useState(null);
     const [uploadDialogOpen, setUploadDialogOpen] = useState(false);
 
+    useEffect(() => {
+        if (tracker.uploads.length > 0) {
```

Comment:
I haven't looked too closely at the uploads code, but I believe this will open the upload dialog with every update to the tracker. So if I drag and drop in 10 items, the dialog will open once for the submission, and then I can close it to work on something else. But I think 10 times after that it will open for each file as they reach the completed state, which could be a bit much.

Reply:
Hmm good catch..It does open multiple times. I think it opens every time a new batch is started (batch size is 3). So I am seeing it open around 3 - 4 times. I figured out a different way to provide feedback to user. I will close this one for now.